### PR TITLE
devtool POC

### PR DESCRIPTION
**Pull request by nhducit**
This is just the PoC to prove that we can reuse react-devtool in angular application.  We still need to polish this PR.

- [x] create a new package for devtool
- [ ] update Readme.md
- [ ] consider using this this [approach](https://netbasal.com/using-react-in-angular-applications-1bb907ecac91)

### CODE REVIEW

**Comment by NetanelBasal** [comment]
Please create a new library in our workspace named devtools. We'll provide it as a separate package named `@ngneat/ng-query-devtools`.

**Comment by nhducit** [comment]
thanks @NetanelBasal I update the description

**Comment by NetanelBasal** [comment]
@nhducit will you make the changes?

**Comment by nhducit** [comment]
Sorry I was busy. Will try to polish this PR tomorrow

File: packages/playground/src/app/app.component.ts [hunk 1]
```diff
@@ -71,4 +73,12 @@ import { RouterModule } from '@angular/router';
     </main>
   `,
 })
-export class AppComponent {}
+export class AppComponent implements AfterViewInit {
+  private queryClient = inject(QUERY_CLIENT);
+  ngAfterViewInit() {
+    console.log('ngAfterViewInit');
+    // Type 'import("/Users/ducnguyen/code/ng-query/node_modules/@tanstack/query-core/build/lib/queryClient").QueryClient' is not assignable to type 'import("/Users/ducnguyen/code/ng-query/node_modules/@tanstack/react-query/node_modules/@tanstack/query-core/build/lib/queryClient").QueryClient'.
```
Comment (nhducit) [inline packages/playground/src/app/app.component.ts:80]: I don't know how to fix this issue yet

**Comment by nhducit** [comment]
hi @NetanelBasal I created a new package for the devtool. I have some questions:
- since the devtool doesn't use any Angular-specific code so I create a new package with @nwrl/js-library schema. Is this ok?

File: packages/ng-query/src/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export * from './lib/query';
+export * from './lib/query-client';
```
Comment (nhducit) [inline packages/ng-query/src/index.ts:2]: remove later

**Comment by NetanelBasal** [comment]
Thanks. I already added it.

**Pull request closed without merging**